webpack.common.js

```diff
@@ -99,6 +102,10 @@ function createConfig(target /*: Target */, dev /*: boolean */,
             new webpack.EnvironmentPlugin({
                 NODE_ENV: dev ? 'development' : 'production',
             }),
+            target.name === 'katex' && new webpack.optimize.CommonsChunkPlugin({
+                name: 'katex',
+                minChunks: Infinity,
+            }),
```

Comment:
So if any modules from the `katex` chunk also appear in any other chunk (e.g. the `contrib/html`) remove them from the other chunk.

Reply:
@kevinbarabash Yes 😄